tests/UnitTests_hera.intel.log

```diff
@@ -1,8 +1,15 @@
 
+<<<<<<< HEAD
```

Comment:
Please resolve this conflict.

Reply:
This will be fixed when I upload the final regression test logs.